src/battle_script_commands.c

```diff
@@ -7680,7 +7714,8 @@ static void Cmd_various(void)
         }
         return;
     case VARIOUS_GRAVITY_ON_AIRBORNE_MONS:
-        if (gStatuses3[gActiveBattler] & STATUS3_ON_AIR)
+        // Cancel all multiturn moves of IN_AIR Pokemon except those being targeted by Sky Drop.
+        if (gStatuses3[gActiveBattler] & STATUS3_ON_AIR && !(gActiveBattler == gBattleStruct->skyDropTargets[1] || gActiveBattler == gBattleStruct->skyDropTargets[3]))
```

Comment:
should be able to just check status3 of `gActiveBattler` and gBattleStruct->skyDropTargets[gActiveBattler], if that field is not 0xFF

Reply:
Gravity will still affect both the target and the attacker, since the target is released and brought down once the attacker goes through CancelMultiTurnMoves. The added condition skips the unnecessary message "{target_of_sky_drop} can't stay airborne because of gravity!" The target's sprite will have already reappeared along with the attacker's, so there's no need for this message.